app/main.py

```diff
@@ -1,8 +1,52 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+
+        self.distance = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def wash_single_car(self, car: Car) -> None:
+        car.clean_mark = self.clean_power
+
+    def calculate_washing_price(self, car: Car) -> float:
+        first_operation = (car.comfort_class
+                           * (self.clean_power - car.clean_mark))
+        second_operation = (first_operation
+                            * (self.average_rating / self.distance))
+
```

Comment:
are the blank lines essential?

Reply:
yes, the blank lines are essential, because otherwise the line is too long and doesn't pass flake8 rules